app/controllers/lit/localizations_controller.rb

```diff
@@ -54,6 +54,7 @@ def find_localization
 
     def after_update_operations
       @localization.update_column :is_changed, true
+      Lit.get_key_value_engine['lit_last_updated_at'] = Time.now.to_i
```

Comment:
You no longer need this value after changes in cache.rb file. Now you just check DB for latest updated_at. I think it should be removed for clarity